src/main/java/picard/sam/SamErrorMetric/CollectSamErrorMetrics.java

```diff
@@ -186,6 +186,11 @@
     )
     public boolean INTERVAL_ITERATOR = false;
 
+    @Argument(shortName = "EXT",
+            doc = "Append the given file extension to all metric file names (ex. OUTPUT.insert_size_metrics.EXT). None if null",
+            optional = true)
+    public String FILE_EXTENSION = null;
```

Comment:
Why not default to "" and then the logic below will be simpler?

Reply:
Resolved.